### PR TITLE
Disabled usage of the POLYGON_MODE_LINE gpu feature in the examples

### LINKED ISSUE #14353: 3D line example broken on web
## Bevy version

Bevy 0.14

## What you did

Go to 3D lines example here: https://bevyengine.org/examples/3d-rendering/lines/

## What went wrong

I only see the gray background and nothing else.

Console log shows a lot of things, I think the relevant part is:

> ERROR  Device::create_render_pipeline error: Features Features(POLYGON_MODE_LINE) are required but not enabled on the device log.target = "wgpu_core::device::global";
> log.module_path = "wgpu_core::device::global";
> log.file = "/Users/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wgpu-core-0.21.1/src/device/global.rs";
> log.line = 1636; [wasm_example.js:435:21](https://bevy-webgl2-examples.pages.dev/3D%20Rendering/lines/wasm_example.js)
> ERROR  Handling wgpu errors as fatal by default log.target = "wgpu::backend::wgpu_core";
> log.module_path = "wgpu::backend::wgpu_core";
> log.file = "/Users/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wgpu-0.20.1/src/backend/wgpu_core.rs";
> log.line = 2995;

I can provide more info if necessary.

## Additional information

Using Firefox 128 on Linux desktop.
Also didn't work on Firefox 128 on Android.

### LINKED ISSUE #14371: 3d shapes web example crashes when enabling wireframes
## Bevy version

https://bevyengine.org/examples/3d-rendering/3d-shapes/

WebGL and WebGPU backends are affected.

## What you did

Press space.

## What went wrong

Example freezes, logs show that the app is panicking.

```
panicked at /Users/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wgpu-0.20.1/src/backend/wgpu_core.rs:2996:5:
wgpu error: Validation Error

Caused by:
    In Device::create_render_pipeline
      note: label = `prepass_pipeline`
    Features Features(POLYGON_MODE_LINE) are required but not enabled on the device
```

### PR DESCRIPTION
**Pull request by SarthakSingh31**
Fixes #14353
Fixes #14371

### CODE REVIEW

**Comment by mockersf** [comment]
Could you also update the 2d_shapes example?

File: examples/3d/3d_shapes.rs [hunk 1]
```diff
@@ -17,10 +18,18 @@ fn main() {
...
+            #[cfg(any(not(target_arch = "wasm32"), feature = "webgpu"))]
             WireframePlugin,
         ))
         .add_systems(Startup, setup)
-        .add_systems(Update, (rotate, toggle_wireframe))
+        .add_systems(
+            Update,
+            (
+                rotate,
+                #[cfg(any(not(target_arch = "wasm32"), feature = "webgpu"))]
```
Comment (mockersf) [inline examples/3d/3d_shapes.rs:29]: why not just exclude wasm32?

Comment (SarthakSingh31) [reply examples/3d/3d_shapes.rs:29]: I thought that `POLYGON_MODE_LINE` was supported in WebGpu but its not. Will remove this.

**Comment by SarthakSingh31** [comment]
> Could you also update the 2d_shapes example?
Updated

**Review by mockersf — APPROVED** [review]
Could you add a comment to the 2d_shapes and 3d_shapes examples explaining that wireframe doesn't work in Wasm?

**Review by mockersf — APPROVED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -760,7 +760,7 @@ doc-scrape-examples = true
 name = "Lines"
 description = "Create a custom material to draw 3d lines"
 category = "3D Rendering"
-wasm = true
+wasm = false
```
Comment (BD103) [inline Cargo.toml:763]:
```suggestion
# WASM does not support the `POLYGON_MODE_LINE` feature.
wasm = false
```

**Review by BD103 — APPROVED** [review]
Looks good! I have one non-blocking request to add a comment, but no worries if you don't get around to it :)

**Pull request merged**